translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+        <target>Votre phalange doit être plus grande que celle qu&apos;elle tente de capturer.</target>
+      </trans-unit>
+      <trans-unit id="2291068586508886218" datatype="html">
+        <source>This square is not aligned with the selected piece.</source>
+        <target>Cette case n&apos;est pas alignée avec la pièce sélectionnée.</target>
+      </trans-unit>
+      <trans-unit id="8716552567618018184" datatype="html">
+        <source>A single piece can only move by one square.</source>
+        <target>Une pièce seule ne peut se déplacer que d&apos;une case.</target>
+      </trans-unit>
+      <trans-unit id="3099022711875888574" datatype="html">
+        <source>A single piece cannot make a capture.</source>
+        <target>Une pièce seule ne peut pas capturer.</target>
+      </trans-unit>
+      <trans-unit id="5151115756771676188" datatype="html">
+        <source>This square is not aligned with the direction of the phalanx.</source>
+        <target>Cette case n&apos;est pas alignée avec la direction de la phalange.</target>
+      </trans-unit>
+      <trans-unit id="5279717712059022209" datatype="html">
+        <source>A phalanx can&apos;t contain pieces outside of the board.</source>
+        <target>Une phalange ne peut pas contenir de pièce hors du plateau.</target>
+      </trans-unit>
+      <trans-unit id="3733956045714659124" datatype="html">
+        <source>A phalanx cannot contain an empty square.</source>
+        <target>Une phalange ne peut pas contenir case vide.</target>
+      </trans-unit>
+      <trans-unit id="8175810329145642315" datatype="html">
+        <source>A phalanx cannot contain an piece of the opponent.</source>
+        <target>Une phalange ne peut pas contenir de pièce de l'adversaire.</target>
+      </trans-unit>
+      <trans-unit id="8733936607898144583" datatype="html">
+        <source>Initial board</source>
+        <target>Plateau initial</target>
+      </trans-unit>
+      <trans-unit id="6855901703200229431" datatype="html">
+        <source>This is the initial board.
+        The top line is the starting line of Light.
+        The bottom line is the starting line of Dark.</source>
+        <target>Ceci est le plateau de départ.
+        La ligne tout en haut est la ligne de départ de Clair.
+        La ligne tout en bas est la ligne de départ de Foncé.</target>
+      </trans-unit>
+      <trans-unit id="6886026531074912078" datatype="html">
+        <source>Goal of the game (1/2)</source>
+        <target>But du jeu (1/2)</target>
+      </trans-unit>
+      <trans-unit id="3201616758404916501" datatype="html">
+        <source>After multiple moves, if at the beginning of its turn, a player has more piece on the opponent&apos;s starting line
+        than the number of pieces the opponent has on the player&apos;s starting line, the player wins.
+        Here, it&apos;s Dark turn to play: Dark has therefore won.</source>
```

Comment:
c'est correct "Dark turn to play" au lieu de juste "Dark turn" ou peut être "Dark turns" ??

Reply:
Changé en la version correcte: Here, it's Dark's turn to play